File: main-api/src/main/java/com/italomonitor/main/dto/response/AgenteResponse.java
package com.italomonitor.main.dto.response;

import lombok.AllArgsConstructor;
import lombok.Builder;
import lombok.Getter;
import lombok.NoArgsConstructor;
import lombok.Setter;

@NoArgsConstructor
@AllArgsConstructor
@Getter
@Setter
@Builder
public class AgenteResponse {

	private Long id;
	
	private String chave;
	
	private String nome;
	
}

File: frontend-app/src/core/model/dto/response/AgenteResponse.ts

export interface AgenteResponse {
    id : number;
    chave : string;
    nome : string;
    dispositivosQuant : number;
}

export const DEFAULT_AGENTE_OBJ : AgenteResponse = {
    id : 0,
    chave : '',
    nome : '',
    dispositivosQuant: 0
};